app/views/alchemy/essences/shared/_essence_picture_tools.html.erb

```diff
@@ -12,7 +12,7 @@
       padding: false
     }, {title: Alchemy.t('Edit Picturemask')} %>
 <%- else -%>
-  <a href="#" class="disabled" tabindex="-1"><%= render_icon(:crop) %></a>
+  <a href="#" class="disabled" onclick="javascript: return false;" tabindex="-1"><%= render_icon(:crop) %></a>
```

Comment:
This change is nice, but does not belong into this commit. Could you please add a dedicated commit for this change? Thanks.